cli/src/commands/migration/status.test.ts

```diff
@@ -0,0 +1,256 @@
+import { Config } from '@oclif/core';
+import fetch from 'node-fetch';
+import { beforeEach, describe, expect, test, vi } from 'vitest';
+import { clearEnvVariables } from '../utils.test.js';
+import prompts from 'prompts';
+import MigrationStatus from './status.js';
+import chalk from 'chalk';
+
+vi.mock('prompts');
+vi.mock('node-fetch');
+vi.mock('fs/promises');
+
+clearEnvVariables();
+
+beforeEach(() => {
+  process.env.XATA_API_KEY = '1234abcdef';
+  process.env.XATA_BRANCH = 'main';
+});
+
+const fetchMock = fetch as unknown as ReturnType<typeof vi.fn>;
+const promptsMock = prompts as unknown as ReturnType<typeof vi.fn>;
+
+const REGION = 'us-east-1';
+const baseUrl = `https://test-1234.${REGION}.xata.sh/db/db1:main`;
+
+export const baseFetch = (url: string, request: any) => {
+  if (url === 'https://api.xata.io/workspaces' && request.method === 'GET') {
+    return {
+      ok: true,
+      json: async () => ({
+        workspaces: [{ id: 'test-1234', name: 'test-1234' }]
+      })
+    };
+  }
+
+  if (url === 'https://api.xata.io/workspaces/test-1234/dbs' && request.method === 'GET') {
+    return {
+      ok: true,
+      json: async () => ({
+        databases: [{ name: 'db1', region: REGION }]
+      })
+    };
+  }
+
+  if (url === `${baseUrl}` && request.method === 'GET') {
+    return {
+      ok: true,
+      json: async () => ({
+        usePgRoll: true,
+        schema: { tables: [{ name: 'table1', columns: [{ name: 'description', type: 'string' }] }] }
+      })
+    };
+  }
+
+  if (url === `${baseUrl}/schema` && request.method === 'GET') {
+    return {
+      ok: true,
+      json: async () => ({
+        schema: {
```

Comment:
Should we share this schema between the new tests? Seems to be the same/very similar